solutions/usaco-860.mdx

```diff
@@ -0,0 +1,130 @@
+---
+id: usaco-860
+source: USACO Silver 2018 December
+title: Mooyo Mooyo
+author: Nathan Gong
+---
+
+[Official Analysis](http://www.usaco.org/current/data/sol_mooyomooyo_silver_dec18.html)
+
+## Implementation
+
+<LanguageSection>
+
+<JavaSection>
+
+**Time Complexity:** $\mathcal{O}(N^2)$
+
+```java
+import java.util.*;
+import java.io.*;
+
+public class MooyoMooyo {
+	static int n, k;
+	static int[][] grid, region;
+	
+	public static void main(String[] args) throws IOException {
+		Kattio io = new Kattio("mooyomooyo");
+
+		n = io.nextInt();
+		k = io.nextInt();
+		grid = new int[n][10];
```

Comment:
magic numbers are bad practice, could you put it in a final variable like `ROW_WIDTH` or something

Reply:
don't think it matters; however, it would be nice if 10 was an appropriately named variable

Reply:
I just named it m because that's pretty standard for usaco